tripal_chado/src/Plugin/TripalImporter/TaxonomyImporter.php

```diff
@@ -55,6 +55,16 @@ class TaxonomyImporter extends ChadoImporterBase implements ContainerFactoryPlug
    */
   protected object $property_buddy;
 
+  /**
+   * Options for file retrieval from NCBI
```

Comment:
Same as what I suggested in the pub med importer.
```suggestion
   * Options for file retrieval from NCBI.
   *  
   * NOTE: NCBI accepts 3 requests/second by default but will allow 
   * 10 requests/second if an API key is provided. This is defined  
   * via the rate_limit key.
```